example_2/description/rviz/diffbot_view.rviz

```diff
@@ -0,0 +1,172 @@
+Panels:
```

Comment:
What is here the difference to the previous file. From your perspective, does it make sense to have two `.rviz` files?

Reply:
I'll check that when I'm back in Austria and having access to any other machine then my macbook M1, did not manage to create a docker with GUI support 👻

Reply:
The only difference: one has `odom` and the other `base_link` as reference frame.

Reply:
OK, I understand. One is for viewing and other is for use with controller. This is fine then.